### PR TITLE
Add mips support for QemuTracerHelper

### PR DESCRIPTION
**Pull request by Sparrrgh**
Add mips support for QemuTracerHelper

### CODE REVIEW

**Comment by domenukk** [comment]
Thank you! Please run `cargo +nightly fmt`

**Comment by Sparrrgh** [comment]
Done!

File: libafl_qemu/src/calls.rs [hunk 1]
```diff
@@ -99,6 +99,13 @@ where
         ret_addr
     };
 
+    #[cfg(any(cpu_target = "mips", cpu_target = "mips"))]
```
Comment (domenukk) [inline libafl_qemu/src/calls.rs:102]: Just `#[cfg(cpu_target = "mips")]` should suffice, right?

Comment (Sparrrgh) [reply libafl_qemu/src/calls.rs:102]: Yes, it is sufficient 😅

Comment (Sparrrgh) [reply libafl_qemu/src/calls.rs:102]: Yes, it doesn't need the `any()` part

**Comment by andreafioraldi** [comment]
Can you please remove also the cfg guarding the calls module? https://github.com/Sparrrgh/LibAFL/blob/main/libafl_qemu/src/lib.rs#L79

**Pull request merged**